src/build_using_cmake_and_homebrew_on_macos.tm

```diff
@@ -0,0 +1,758 @@
+      up-to-date
+
+      To reinstall 4.8.7_6, run `brew reinstall qt@4`
+    </unfolded-io>
+
+    <\unfolded-io|Shell] >
+      <code*|/usr/local/Cellar/qt@4/4.8.7_6/bin/qmake> --version
+    <|unfolded-io>
+      /usr/local/Cellar/qt@4/4.8.7_6/bin/qmake --version
+
+      QMake version 2.01a
+
+      Using Qt version 4.8.7 in /usr/local/lib
+    </unfolded-io>
+
+    <\input|Shell] >
+      \;
+    </input>
+  </session>
+
+  <paragraph|Switch between Qt 4 and Qt 5>
+
+  Sometimes, we need to keep both Qt 4 and Qt 5 installed by homebrew. To
+  switch between Qt 4 and Qt 5:
+
+  <\session|shell|default>
+    <\unfolded-io|Shell] >
+      qmake --version
+    <|unfolded-io>
+      qmake --version
+
+      QMake version 2.01a
+
+      Using Qt version 4.8.7 in /usr/local/lib
+    </unfolded-io>
+
+    <\unfolded-io|Shell] >
+      brew unlink qt@4 && brew link qt --force
+    <|unfolded-io>
+      brew unlink qt@4 && brew link qt --force
+
+      Unlinking /usr/local/Cellar/qt@4/4.8.7_6... 666 symlinks removed
+
+      Linking /usr/local/Cellar/qt/5.15.0... 510 symlinks created
+
+      \;
+
+      If you need to have this software first in your PATH instead consider
+      running:
+
+      \ \ echo 'export PATH="/usr/local/opt/qt/bin:$PATH"' \<gtr\>\<gtr\>
+      ~/.zshrc
+    </unfolded-io>
+
+    <\unfolded-io|Shell] >
+      qmake --version
+    <|unfolded-io>
+      brew link qt --force
+
+      Linking /usr/local/Cellar/qt/5.15.0... 510 symlinks created
+
+      \;
+
+      If you need to have this software first in your PATH instead consider
+      running:
+
+      \ \ echo 'export PATH="/usr/local/opt/qt/bin:$PATH"' \<gtr\>\<gtr\>
+      ~/.zshrc
+    </unfolded-io>
+
+    <\unfolded-io|Shell] >
+      brew unlink qt && brew link qt@4
+    <|unfolded-io>
+      brew unlink qt && brew link qt@4
+
+      Unlinking /usr/local/Cellar/qt/5.15.0... 510 symlinks removed
+
+      Linking /usr/local/Cellar/qt@4/4.8.7_6... 666 symlinks created
+    </unfolded-io>
+
+    <\unfolded-io|Shell] >
+      qmake --version
+    <|unfolded-io>
+      qmake --version
+
+      QMake version 2.01a
+
+      Using Qt version 4.8.7 in /usr/local/lib
+    </unfolded-io>
+
+    <\input|Shell] >
+      \;
+    </input>
+  </session>
+
+  If you have trouble on <shell|brew link qt --force> and <shell|brew unlink
+  qt>. Make sure that you have installed Qt before using homebrew:
+
+  <\session|shell|default>
+    <\input>
+      Shell]\ 
+    <|input>
+      brew install qt
+    </input>
+  </session>
+
+  <subsection*|Install Ghostscript>
+
+  <\session|shell|default>
+    <\input>
+      Shell]\ 
+    <|input>
+      brew install ghostscript
+    </input>
+
+    <\unfolded-io|Shell] >
+      gs --version
+    <|unfolded-io>
+      gs --version
+
+      9.52
+    </unfolded-io>
+
+    <\input|Shell] >
+      \;
+    </input>
+  </session>
+
+  <subsection*|Missing dependencies>
+
+  Please tell me by creating PR for any missing dependencies.
```

Comment:
You name never appears, so the user do not know what to do here. Maybe be more explicit where to ask for help or submit improvements.

Reply:
updated